docs/tutorials/05-model-monitoring.ipynb

```diff
@@ -78,13 +80,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Set datastore profiles and enable model monitoring\n",
+    "### Set datastore profiles \n",
     "\n",
-    "Before you enable model monitoring, you must configure datastore profiles for TSDB and streaming endpoints. A [datastore profile](https://docs.mlrun.org/en/stable/store/datastore.html#data-store-profiles) holds all the information required to address an external data source, including credentials.\n",
+    "A [datastore profile](https://docs.mlrun.org/en/stable/store/datastore.html#data-store-profiles) holds all the information required to address an external data source, including credentials. Before you enable model monitoring, you must configure datastore profiles for TSDB and streaming endpoints:\n",
+    "- TSDB (time series database) stores, retrieves, and analyzes time-stamped data.\n",
+    "- Streaming platforms are used for delivering data continuously, for example data constantly arriving from different sources that needs to be stored or analyzed.\n",
     "\n",
     "Model monitoring supports [Kafka](https://docs.mlrun.org/en/stable/api/mlrun.datastore/index.html#mlrun.datastore.datastore_profile.DatastoreProfileKafkaSource) and [V3IO](https://docs.mlrun.org/en/stable/api/mlrun.datastore/index.html#mlrun.datastore.datastore_profile.DatastoreProfileV3io) as streaming platforms, and [TDEngine](https://docs.mlrun.org/en/stable/api/mlrun.datastore/index.html#mlrun.datastore.datastore_profile.TDEngineDatastoreProfile) and [V3IO](https://docs.mlrun.org/en/stable/api/mlrun.datastore/index.html#mlrun.datastore.datastore_profile.DatastoreProfileV3io) as TSDB platforms. \n",
     " \n",
-    "See [set_model_monitoring_credentials](https://docs.mlrun.org/en/stable/api/mlrun.projects/index.html#mlrun.projects.MlrunProject.set_model_monitoring_credentials) for more details."
+    "See [set_model_monitoring_credentials](https://docs.mlrun.org/en/stable/api/mlrun.projects/index.html#mlrun.projects.MlrunProject.set_model_monitoring_credentials) for more details.\n",
```

Comment:
It feels a bit out of context here. We tell the user "see set_model..." but without explain what is it and moreover we don't use it directly in this tutorial... 
I’d just explain that once you’ve configured and saved the datastore profiles under your project, you need to call the `set_model_monitoring_credentials` API to register them with the model monitoring resources. In this tutorial we already wrapped this API under the helper `enbale_model_monitoring` that can be found in the `src.model_monitoring_utils`.